File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.ts
function removeDuplicates(nums: number[]): number {
    var count = 1, k = 1;
    for(var i=1; i<nums.length; i++){
        if(nums[i] == nums[i-1]){
            count += 1;
        } else{
            count = 1;
        }
        if(count < 3){
            nums[k] = nums[i];
            k++;
        }
    }
    return k;
};

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count = 1, j=1;
        int size = nums.size();
        for (int i = 1; i < size; i++) {
            if(nums[i] == nums[i-1]) {
                count++;
            }
            else{
                count = 1;
            }
            if(count < 3){
                   nums[j] = nums[i];
                   j++; 
                }
        }
        return j;
    }
};
